frontend/src/components/ui/AppHelp.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="instructions">
+    <a href="#" id="show-modal" @click="showModal = true">Instructions</a>
+  </div>
+  <Teleport to="body">
+    <!-- use the modal component, pass in the prop -->
+    <AppModal :show="showModal" @close="showModal = false">
+      <template #header>
```

Comment:
should we be consistent with how we specify the slot that a template is for?
Using # here but v-slot:'' elsewhere